modules/local/faitochr/main.nf

```diff
@@ -0,0 +1,49 @@
+process FAITOCHR {
+    tag "$meta.id"
+    label 'process_single'
+
+    input:
```

Comment:
no container/conda declaration?

Reply:
It's just using awk.
Is there a container for that ?

Reply:
@maxulysse There is the gawk conda forge package but i'm not sure it's necessary.